dll/win32/shell32/folders/CRecycleBin.cpp

```diff
@@ -883,12 +889,16 @@ TRASH_CanTrashFile(LPCWSTR wszPath)
         return FALSE;
     }
 
-    // Only keep the base path.
+    // Create a wide character buffer for our path copy
```

Comment:
No need to add a comment line for each instruction.
Just say that you are going to retrieve the root path of the original path.
Then remove the comment before StringCbCopy and PathStripToRootW calls.